etc/config/components/airflow/slack_operator.py

```diff
@@ -26,66 +26,6 @@
 from airflow.exceptions import AirflowException
 
 
-class SlackAPIOperator(BaseOperator):
```

Comment:
I'm not sure how/when these classes get instantiated, but it seems that if the `BaseOperator` needed to be imported previously, then `SlackAPIOperator`'s import would be necessary for this change.  Looks like `SlackHook` and `AirflowException` imports can also be removed.

Reply:
These classes arent instantiated locally, if I understand correctly, more being used as a source of information/data to populate the component palette properties and to construct the DAG using the airflow template. As long as the operators are defined correctly on the deployment side, where they are instantiated, a local change here should be ok

Reply:
Are you saying that, any customizations we make locally are lost because the server side installed component is used instead? Can we then actually just query which components are available in the server?

Reply:
Sort of.
We can make for lack of a better term `cosmetic` modifications like this removal because all we are interested in right now are the `arguments` needed to make the operator function. This in turn is the next item in question, that the `arguments` we are interested in, HAVE to be aligned and the same as the operator being used in the deployment.  So any "customizations" will need to fit within the confines of the source operator class definition in the airflow deployment, or included in the DAG itself.

Reply:
yeah, that is sort of thing we need to further discuss/investigate in the upcoming releases.

Reply:
I still feel, for the sake of correctness and good example, these _cosmetic_ modifications should compile if their original incantations compile.

Reply:
opened two issues for this #2008 and #2009